tests/L0/run_fused_layer_norm/test_fused_layer_norm.py

```diff
@@ -8,291 +8,282 @@
 from apex.normalization import MixedFusedLayerNorm
 from apex.normalization import MixedFusedRMSNorm
 
+from torch.testing._internal import common_utils
+from torch.testing._internal.common_device_type import instantiate_device_type_tests
+from torch.testing._internal.common_device_type import onlyCUDA
```

Comment:
Is this used somewhere? `instantiate_device_type_tests` would be enough
```suggestion

```

Reply:
Fixed